frontend/src/shared/ui/input/input.module.css

```diff
@@ -0,0 +1,141 @@
+.input__wrapper {
+  --border-color: var(--outline-color);
+
+  position: relative;
```

Comment:
Maybe it's better to use `.scss` at this beginning state?

Reply:
Yea, we can, I think we need to merge each open PR and re-write all `css` files using scss